src/utils/AxiosFactory.ts

```diff
@@ -30,22 +30,32 @@ export default class AxiosFactory {
     if (!axiosInstance) {
       // Create
       axiosInstance = axios.create(instanceConfiguration.axiosConfig);
-      // Add a Request interceptor
-      axiosInstance.interceptors.request.use((request: AxiosRequestConfig) => {
-        Logging.logAxiosRequest(tenantID, request);
-        return request;
-      }, async (error: AxiosError) => {
-        Logging.logAxiosError(tenantID, error);
-        return Promise.reject(error);
-      });
-      // Add a Response interceptor
-      axiosInstance.interceptors.response.use((response: AxiosResponse) => {
-        Logging.logAxiosResponse(tenantID, response);
-        return response;
-      }, async (error: AxiosError) => {
-        Logging.logAxiosError(tenantID, error);
-        return Promise.reject(error);
-      });
+
+      // FIXME
```

Comment:
Add comments at the calling site. The interface is clean here. You just have to explain why at the call site you disable interceptors.

Reply:
done